doc/conf.py

```diff
@@ -207,6 +207,9 @@ def inner(fn, *iargs, **ikwargs):
 
 # -- General Configuration -----------------------------------------------------
 
+# Set a var if we're building docs for the live site or not
+on_saltstack = 'SALT_ON_SALTSTACK' in os.environ
+
 project = 'Salt'
 copyright = '2016 SaltStack, Inc.'
```

Comment:
It would be nice if we could dynamically update the year using the same technique as we do for the `today` variable below.